docs/guides/components/page_break.rst

```diff
@@ -2,6 +2,21 @@ PageBreak
 =========
 For use in forms which are rendered to PDF to indicate that a page break is required.
 
-The ``margin_top`` property is optional. In the generated pdf you may want more or less white space at the top of the page.
+The optional ``margin_top`` property  allows to change the amount of white space at the top of the page.
 You can set the ``margin_top`` property to a positive/negative number to adjust the whitespace.
-Most of the time this is unnecessary. This won't have any effect on the designer only the generated pdf.
+Most of the time this is unnecessary. This won't have any effect on the designer, only the generated PDF.
+
+The optional ``border`` property allows to define the style of the component in the IDE.
```

Comment:
suggestion:

```text
The optional ``border`` property defines the style of the PageBreak.
```

Reply:
I think I have addressed all your points.

I also have addressed https://github.com/anvilistas/anvil-extras/issues/142, so I could test it.